CMakeLists.txt

```diff
@@ -442,12 +443,14 @@ add_dependencies(check_lcf test_runner_lcf)
 add_dependencies(check check_lcf)
 
 # benchmarks
-file(GLOB BENCH_FILES ${CMAKE_CURRENT_SOURCE_DIR}/bench/*.cpp)
-foreach(i ${BENCH_FILES})
-	get_filename_component(name "${i}" NAME_WE)
-	add_executable(bench_${name} ${i})
-	target_link_libraries(bench_${name} lcf)
-endforeach()
+if(NOT DISABLE_BENCHMARKS)
```

Comment:
Should we call this `LIBLCF_ENABLE_BENCHMARKS` to match player?

Reply:
I consider changing all the options to ``LCF_SOMETHING`` like we have in Player and deprecating all the ``DISABLE`` options.